VibeSomeMauiBro/Services/CatService.cs

```diff
@@ -14,10 +14,11 @@ public interface ICatService
 public class CatService : ICatService
 {
     private readonly HttpClient _httpClient;
-    private readonly HashSet<Cat> _likedCats = new();
+    private readonly List<Cat> _likedCats = new();
```

Comment:
Don't change this type from HashSet.

Call `.ToList()` if you need a List<T> when persisting to disk.